### ISSUE TITLE
Marlin bugfix 1.1x BLtouch problem

### ISSUE THREAD

**Issue by tankdoc** [MarlinFirmware/Marlin#10692]
Hi, there!

I made a large 3d printer to print a big size prototype, but I faced problem.

I used BLtouch to verify the angle of the bed, but when I select the 'level bed', the movement is weird.

First, the nozzle goes home and get the bed data, it moves to next X coordinate.

But when goes to next coordinate, it doesn't move and makes awful noise.

To prevent that, I changed endstop settings and other problems.

But well, same problem.

The jog mode can moves it, but level bed function doesn't move.

If someone have a solution, Please anybody help.

Thanks

**Comment by boelle** [comment]
is the problem the same with bugfix 2.0 ?

**Comment by tankdoc** [comment]
well I dunno the problem occurred at bugfix 2.0, but i think that problem is same.
the program is perfect, but.......
my customization doesn't work.
I installed the BLtouch option to level my large bed.
Also followed the tutorial that the inventor has noticed.
However, the level probing through the X axis doesn't work.

In a nutshell, probing along the Y axis works well, but probing along X axis has error with the second X coordinate. I need to solve this problem.

P.S.
I'm so tired of this.......
till now, i revising my code comparing with the Black widow 3d printer configuration.

**Comment by thinkyhead** [comment]
Perhaps your Y max feedrate is too high. You haven't described your hardware or provided any data.

Please ZIP up your `Configuration.h` and `Configuration_adv.h` files (as requested in the Issue template) and drop them into your next reply. We'll check them over and see if anything is amiss.

**Comment by tankdoc** [comment]
Woah.

I didn't think I could get assistance that quickly.

I really appreciate to everyone.

Just hold up, and I will reply my configuration.h and configuration_adv.h file.

P.S.

This printer has 1200mm x 1200mm bed size and 600mm printable height.

**Comment by tankdoc** [comment]
Here. there is my configuration and _adv.h files
[configuration and its adv.zip](https://github.com/MarlinFirmware/Marlin/files/1997289/configuration.and.its.adv.zip)

**Comment by thinkyhead** [comment]
Your max feedrate on X and Y of 500mm/s is way too high, especially when you have 1600 steps per mm on the X and Y axes. Attempting to do 800,000 pulses per second will just reboot your board.

Reduce your max feedrates in the configs. You can find
reasonable limits by experimenting with `M203 Xnnn Ynnn`.

**Comment by boelle** [comment]
@tankdoc did you try what thinky suggested?

**Comment by tankdoc** [comment]
@boelle
Yeah. I did it long time ago.
But... It didn't work.
So.........
I pended the project.
Sooner or later, i gonna fix that problem.
P.S. this giant printer(I named it AL. Monger) has been taken too many space to our lab.
 So i have searched many information to change it differently.
 That prototype became so rusty(Linear motion guide) and dusty.
 After graduating the master student course, i will make it work like bigrep.
Thanks to reply to me.
With fingers crossed,
Tankdoc.
